src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('.population')];
+
+const num = Array.from(population).map((ele) => {
+  const txt = ele.textContent.replace(/,/g, '');
```

Comment:
Ensure that the text content of the elements is always a valid number after removing commas. If there's any chance of invalid data, consider adding error handling or validation.